Workbooks/Workloads/Enable monitoring group/Enable monitoring group.workbook

```diff
@@ -141,7 +141,7 @@
+        "query": "Resources\r\n| where id in~ ({VmsWithTag})\r\n| project ['Virtual machine'] = tolower(id), Name = name, ['Resource group'] = strcat('/subscriptions/', subscriptionId, '/resourceGroups/', resourceGroup), Subscription = subscriptionId, Location = location,  OsType = iff(properties.osProfile.windowsConfiguration == '', 'Linux', 'Windows'), Offer = tostring(properties.storageProfile.imageReference.offer)\r\n//| where Offer == 'UbuntuServer' //and Location == 'centralus'\r\n| join kind = leftouter (Resources\r\n    | where type =~ 'microsoft.compute/virtualmachines/extensions' and properties.type == 'OmsAgentForLinux' and properties.publisher == 'Microsoft.EnterpriseCloud.Monitoring'\r\n    | project Vm = extract(@'(.+)/extensions/.+', 1, tolower(id)), WorkspaceId = tostring(properties.settings.workspaceId)\r\n    | where Vm != ''\r\n    ) on $left.['Virtual machine'] == $right.Vm\r\n| join kind = leftouter (Resources\r\n    | where type =~ 'microsoft.compute/virtualmachines/extensions' and name =~ 'workload.linuxconfigagent'\r\n    | project Vm = extract(@'(.+)/extensions/.+', 1, tolower(id)), Insight = tostring(tags[\"monitoring-insight\"])\r\n    | where Vm != ''\r\n    ) on $left.['Virtual machine'] == $right.Vm\r\n| project-away Vm, Vm1\r\n| join kind = leftouter (Resources\r\n| where type =~ 'microsoft.operationalinsights/workspaces'\r\n| project Workspace = id, location, WorkspaceId = tostring(properties.customerId)) on WorkspaceId\r\n| project ['Virtual machine'], Name, ['Resource group'], Subscription, Workspace, Location, State = case(Workspace == '' or Location !in ('eastus', 'centralus', 'westus', 'eastus2', 'westus2', 'eastasia', 'southeastasia', 'northeurope', 'westeurope', 'francecentral', 'uksouth', 'ukwest', 'centralindia', 'japanwest') or Offer != 'UbuntuServer', 'Not supported', Insight == '{WorkloadTag}', 'Monitored', 'Not Monitored'),  Action = case(Workspace == '' or Location !in ('eastus', 'centralus', 'westus', 'eastus2', 'westus2', 'eastasia', 'southeastasia', 'northeurope', 'westeurope', 'francecentral', 'uksouth', 'ukwest', 'centralindia', 'japanwest') or Offer != 'UbuntuServer', 'Not supported', Insight == '{WorkloadTag}', 'Update settings', Insight != '{WorkloadTag}', 'Enable monitoring', ''), ['Enabled insights'] = Insight, Template = case(Workspace == '' or Location !in ('eastus', 'centralus', 'westus', 'eastus2', 'westus2', 'eastasia', 'southeastasia', 'northeurope', 'westeurope', 'francecentral', 'uksouth', 'ukwest', 'centralindia', 'japanwest') or Offer != 'UbuntuServer', 'Community-Workbooks/Workloads/Monitoring checklist', '{OnboardingTemplate}')\r\n| extend Rank = case(Action == 'Update settings', 0, Action == 'Enable monitoring', 1, 2) \r\n| order by Rank asc, Name asc\r\n| project-away Rank\r\n| extend Troubleshooting = iff([\"Enabled insights\"] == \"{WorkloadTag}\", \"Collection logs\", \"\")",

```

Comment:
You may want to do a case insensitive check (=~) for publisher and type.

Reply:
We probably want a better way to manage the list of supported regions across all these templates.